xds/src/main/java/io/grpc/xds/ThreadSafeRandom.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import java.util.concurrent.ThreadLocalRandom;
+import javax.annotation.concurrent.ThreadSafe;
+
+@ThreadSafe // Except for impls/mocks in tests
+interface ThreadSafeRandom {
```

Comment:
Can we do:

```
final class ThreadSafeRandom extends Random {
   @Override
    public nextInt(int n) {
        ThreadLocalRandom.current().nextInt(n);
    }
}
```

and in tests, do similar to create `FakeRandom`. So we would not introduce a new interface.